addons/html_editor/static/src/main/link/link_plugin.js

```diff
@@ -512,6 +514,17 @@ export class LinkPlugin extends Plugin {
                 this.LinkPopoverState.editing = false;
             }
 
+            if (
+                this.linkElement &&
+                this.linkElement.classList.contains("o_link_in_selection") &&
+                !(
+                    this.linkElement.textContent.startsWith("\uFEFF") &&
+                    this.linkElement.textContent.endsWith("\uFEFF")
+                )
+            ) {
+                this.dependencies.linkSelection.padLinkWithZwnbsp(this.linkElement);
```

Comment:
Can't you call `padLinkWithZwnbsp` without having to check beforehand? I had a quick look at the code and it doesn't create new feffs if they already exist.